pkg/resolver/searchComplete.go

```diff
@@ -44,13 +45,21 @@ func SearchComplete(ctx context.Context, property string, srchInput *model.Searc
 	if userDataErr != nil {
 		return []*string{}, userDataErr
 	}
+
+	//check that shared cache has resource datatypes:
+	propTypesCache, err := rbac.GetCache().GetPropertyTypes(ctx, false)
```

Comment:
I'd rename `propTypesCache` to `propTypes` or `propTypesMap` because at this point it's no longer part of the cache.